app/bundles/UserBundle/Translations/en_US/flashes.ini

```
@@ -13,4 +13,4 @@ mautic.user.user.notice.passwordreset="A new password has been generated and wil
 mautic.user.user.notice.passwordreset.success="If this user exists, you have successfully reset your password. You may now login."
 mautic.user.user.notice.passwordreset.error="We encountered an unexpected error while sending the password reset instructions to your email. Please try again later."
 mautic.user.user.notice.passwordreset.missingtoken="Your reset request is missing the security token. Please click the link in your email."
-mautic.user.security.saml.clearsession="An authentication error occurred, please try logging in again. If the issue persists, contact your Campaign Studio Administrator."
+mautic.user.security.saml.clearsession="An authentication error occurred. Please try logging in again. If the issue persists, contact your Campaign Studio Administrator."
```

```suggestion
mautic.user.security.saml.clearsession="An authentication error occurred. Please try logging in again. If the issue persists, contact your Mautic Administrator."
```

Whoops. looks like a proprietary language string snuck in here. cc @escopecz in case you need to make changes in ACS language strings.